app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Api
+  module V1
+    class UsersController < ApplicationController
+      def show
+        user = User.find(params[:id])
+        render json: user
+      end
+
+      def create
+        user = User.new(user_params)
+
+        if user.save
+          render json: user, status: :created
+        else
+          render json: { errors: user.errors.full_messages }, status: :unprocessable_entity
+        end
+      end
+
+      def upload_avatar
+        user = User.find(params[:id])
+        result = Avatars::Upload.call(user: user, avatar: avatar_params[:avatar],
+                                      avatar_url: avatar_params[:avatar_url])
```

Comment:
Исключительно нейминг сервиса

```suggestion
        result = Users::UploadAvatar.call(user: user, avatar: avatar_params[:avatar],
                                      avatar_url: avatar_params[:avatar_url])
```